schema/bom-1.4-SNAPSHOT.proto

```diff
@@ -450,3 +452,179 @@ message Evidence {
+  optional ImpactAnalysisJustification justification = 2;
+  // A response to the vulnerability by the manufacturer, supplier, or project responsible for the affected component or service. More than one response is allowed. Responses are strongly encouraged for vulnerabilities where the analysis state is exploitable.
+  repeated VulnerabilityResponse response = 3;
+  // Detailed description of the impact including methods used during assessment. If a vulnerability is not exploitable, this field should include specific details on why the component or service is not impacted by this vulnerability.
+  optional string detail = 4;
+}
+
+enum ImpactAnalysisState {
+  // The vulnerability has been remediated.
+  IMPACT_ANALYSIS_STATE_RESOLVED = 0;
+  // The vulnerability has been remediated and evidence of the changes are provided in the affected components pedigree containing verifiable commit history and/or diff(s).
+  IMPACT_ANALYSIS_STATE_RESOLVED_WITH_PEDIGREE = 1;
+  // The vulnerability may be directly or indirectly exploitable.
+  IMPACT_ANALYSIS_STATE_EXPLOITABLE = 2;
+  // The vulnerability is being investigated.
+  IMPACT_ANALYSIS_STATE_IN_TRIAGE = 3;
+  // The vulnerability is not specific to the component or service and was falsely identified or associated.
+  IMPACT_ANALYSIS_STATE_FALSE_POSITIVE = 4;
+  // The component or service is not affected by the vulnerability. Justification should be specified for all not_affected cases.
+  IMPACT_ANALYSIS_STATE_NOT_AFFECTED = 5;
+}
+
+enum ImpactAnalysisJustification {
+  // The code has been removed or tree-shaked.
+  IMPACT_ANALYSIS_JUSTIFICATION_CODE_NOT_PRESENT = 0;
+  // The vulnerable code is not invoked at runtime.
+  IMPACT_ANALYSIS_JUSTIFICATION_CODE_NOT_REACHABLE = 1;
+  // Exploitability requires a configurable option to be set/unset.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_CONFIGURATION = 2;
+  // Exploitability requires a dependency that is not present.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_DEPENDENCY = 3;
+  // Exploitability requires a certain environment which is not present.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_ENVIRONMENT = 4;
+  // Exploitability requires a compiler flag to be set/unset.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_BY_COMPILER = 5;
+  // Exploits are prevented at runtime.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_AT_RUNTIME = 6;
+  // Attacks are blocked at physical, logical, or network perimeter.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_AT_PERIMETER = 7;
+  // Preventative measures have been implemented that reduce the likelihood and/or impact of the vulnerability.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_BY_MITIGATING_CONTROL = 8;
+}
+
+enum VulnerabilityResponse {
+  VULNERABILITY_RESPONSE_CAN_NOT_FIX = 0;
+  VULNERABILITY_RESPONSE_WILL_NOT_FIX = 1;
+  VULNERABILITY_RESPONSE_UPDATE = 2;
+  VULNERABILITY_RESPONSE_ROLLBACK = 3;
+  VULNERABILITY_RESPONSE_WORKAROUND_AVAILABLE = 4;
+}
```

Comment:
More default "null" values suggested

Reply:
Changed in c85bdd66a3fe7413bdc31c73dd9347acdb01860d